test/dummy/app/assets/stylesheets/_variables.sass

```diff
@@ -7,7 +7,7 @@ $grid-gutter-width: 4 * $grid-gutter-base
 $grid-gutter-half: $grid-gutter-width / 2
 
 $spacer: $grid-gutter-half
-$spacers: ( 0: 0, 1: $spacer * .25, 2: $spacer * .5, 3: $spacer, 4: $spacer * 1.5, 5: $spacer * 3, h: $grid-gutter-half, g: $grid-gutter-width, gg: 2 * $grid-gutter-width, 40: px-to-rem(40px) )
+$spacers: ( 0: 0, 1: $spacer * .25, 2: $spacer * .5, 3: $spacer * .75, 4: $spacer, 5: $spacer * 1.5, 6: $spacer * 3, h: $grid-gutter-half, g: $grid-gutter-width, gg: 2 * $grid-gutter-width, 40: px-to-rem(40px) )
```

Comment:
3ku prosim nechme jako `$spacer`, ne jako 75% z nej. Je to dlouhodoby bootstrap default, ktery bych nemenil.

Reply:
Ok, vratil jsem zpet.
Pridal jsem tech 12px na konec `$spacers`. Je to takhle ok?